src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+M['audit_log.spaces'] = format_text([[
+    The array of space names for which data operation events (`space_select`,
+    `space_insert`, `space_replace`, `space_delete`) should be logged. The array
+    accepts string values. If set to box.NULL, the data operation events are
+    logged for all spaces.
+]])
+
+M['audit_log.spaces.*'] = format_text([[
+    A specific space name in the array for which data operation events are
+    logged. Each entry must be a string representing the name of the space
+    to monitor.
+
+    Example:
+
+    `spaces: [bands, singers]`, only the events of `bands` and `singers` spaces
+    are logged.
+]])
+
+M['audit_log.to'] = 'Enable audit logging and define the log location.'
+
+M['audit_log.syslog'] = format_text([[
+    This module allows configuring the system logger (syslog) for audit
+    logs in Tarantool. It provides options for specifying the syslog server,
+    facility, and identity for logging messages.
+]])
+
+M['audit_log.syslog.facility'] = format_text([[
+    Specify a system logger keyword that tells syslogd where to send the
+    message. You can enable logging to a system logger using the
+    `audit_log.to` option.
+]])
+
+M['audit_log.syslog.identity'] = format_text([[
+    Specify an application name to show in logs. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+M['audit_log.syslog.server'] = format_text([[
+    Set a location for the syslog server. It can be a Unix socket path
+    starting with "unix:" or an ipv4 port number. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+-- }}} audit_log configuration
+
+-- {{{ compat configuration
+
+M['compat'] = format_text([[
+    The `compat` section defines values of the compat module options.
+]])
+
+M['compat.binary_data_decoding'] = format_text([[
+    Define how to store binary data fields in Lua after decoding:
+
+    - `new`: as varbinary objects
+    - `old`: as plain strings
+]])
+
+M['compat.box_cfg_replication_sync_timeout'] = format_text([[
+    Set a default replication sync timeout:
+
+    - `new`: 0
+    - `old`: 300 seconds
+]])
+
+M['compat.box_consider_system_spaces_synchronous'] = format_text([[
+    Controls whether to consider system spaces synchronous when the
+    synchronous queue is claimed, regardless of the user-provided `is_sync`
+    option. Either enables synchronous replication for system spaces when the
+    synchronous queue is claimed, overriding the user-provided 'is_sync' space
+    option, or falls back to the user-provided 'is_sync' space option.
+]])
+
+M['compat.box_error_serialize_verbose'] = format_text([[
+    Set the verbosity of error objects serialization:
+
+    - `new`: serialize the error message together with other potentially
+    useful fields
+    - `old`: serialize only the error message
+]])
+
+M['compat.box_error_unpack_type_and_code'] = format_text([[
+    Whether to show error fields in `box.error.unpack()`:
```

Comment:
I would say 'all the error fields' in place of 'error fields'. Now it sounds like neither of the fields are shown in one of the modes.

Reply:
Ok. Let's do it this way.